addons/html_editor/static/src/others/embedded_components/core/table_of_content/table_of_content.js

```diff
@@ -9,7 +9,7 @@ export class EmbeddedTableOfContentComponent extends Component {
     };
 
     setup() {
-        this.state = useState({ toc: this.props.manager.structure });
+        this.state = useState({ toc: this.props.manager.structure, open: true });
```

Comment:
I'd use `folded` instead of `open` (values should be inverted). `folded` carries the meaning that it's a boolean, and it describes more precisely what happens to the component if changed